src/templates/icon.overrides.liquid

```diff
@@ -1,3 +1,33 @@
 /*******************************
-         Theme Overrides
+Theme Overrides
 *******************************/
+
+/* Step 1: Common Duotone Properties: All required to make icons render reliably */
+i.icon.duotone {
+  font-family: @duotoneFontName;
+  position: relative;
+  padding-left: 1em; /* make space for the width of the absolutely positioned icon */
+}
+
+/* Step 2: Set the font-family and font-weight for this style */
+i.icon.duotone::before,
+i.icon.duotone::after {
+  /* position both layers of the icon to the left, set our fixed-width width, horizontally center layers, and then vertically align them so they flex with different line heights */
+  position: absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  text-align: center;
+}
+
+/* Step 3: Set the default opacity levels and colors for each layer */
+i.icon.duotone::before {
+  color: var(--fa-primary-color, inherit);
```

Comment:
How does this work in FUI style like `green icon`? How to supply the second color?
There is something like `green-orange` classnaming logic needed instead of the fixed css variables.
Instead selectors like 
```css
/* Make sure to respect the `variation.variables` and `colors.less` file values when generating this*/
i.icon[class^="yellow-"]::before {
  color: yellow;
}
i.icon[class^="-green"]::after {
  color: green;
}
```
would be needed.

This works out as

```html
<i class="yellow-green camera-duotone icon"></i>
```

As duotone is pro plan only, those could be added to the override file as well (as it wont be part of fui core)
For the same reason, i would put the generation inside the overrides file as well.

```less
/* Yes, this _could_ be supported inside the existing generateIcons Funtion.... :) */

.generateSecondaryIcons(@map, @fontFamily) {
  each(@map,{
    @escapedKey: replace(@key,'^([0-9])','\3$1 ');
    @normalizedKey: replace(@escapedKey,'_','.','g');
    i.icon.@{normalizedKey}::after {
      content: "@{value}";
      & when not (@fontFamily = false) {
        font-family: @fontFamily;
      }
    }
  });
};

.generateIcons(@icon-duotone-map,false);
.generateSecondaryIcons(@icon-duotone-secondary-map,false);
.generateIcons(@icon-duotone-aliases-map,false);
.generateSecondaryIcons(@icon-duotone-secondary-aliases-map,false);
```

Reply:
Ok so IIUC we could use class names like green-orange to specify the two icons colours. And we want to support all color variables right?

Concerning the generation I noticed that Thin, which are pro icons too, are not in override file. Shouldn't they be together?

Reply:
I agree, this is inconsistent...alright, lets move this part to the FUI core in a separate PR for FUI. As long as the default duotone maps are empty , nothing is being generated just as for thin. Good point.

Reply:
`i.icon[class^="yellow-"]::before` will only match if the color is the first class of the icon. I suggest to change it to `i.icon[class~="yellow-"]::before` to match anywhere in the class list. What do you think?

Reply:
> `i.icon[class^="yellow-"]::before` will only match if the color is the first class of the icon. I suggest to change it to `i.icon[class~="yellow-"]::before` to match anywhere in the class list. What do you think?

👍🏼 Yes, my bad, that was my intention, so ~= is the way to go

Reply:
Ah damn hyphen does not work with [attr~=value]. Any suggestion?

Reply:
`black-orange` is a single word so [class~='black-'] does not match

Reply:
We could define all combinations if done in a less template and use "black orange" instead

Reply:
[class*="yellow-"] works
https://jsfiddle.net/lubber/Ltzouv8k/5/

Reply:
Of course, damn!